ccp/modules/login-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "3.7"
+
+services:
+  ############################################ Keycloak
+  login-db:
+    image: postgres:15.1-alpine
+    container_name: bridgehead-login-db
+    environment:
+      POSTGRES_USER: "keycloak"
+      POSTGRES_PASSWORD: "${KEYCLOAK_DB_PASSWORD}" # Set in teiler-setup.sh
+      POSTGRES_DB: "keycloak"
+    volumes:
+      - "bridgehead-login-db:/var/lib/postgresql/data"
+
+  login:
+    image: docker.verbis.dkfz.de/ccp/dktk-keycloak:latest
+    container_name: bridgehead-login
+    environment:
+      KEYCLOAK_ADMIN: "admin"
+      KEYCLOAK_ADMIN_PASSWORD: "${KEYCLOAK_ADMIN_PASSWORD}"
```

Comment:
Where is this variable defined?

Reply:
This should be defined in the ccp.conf file